lib/checks/aria/braille-roledescription-equivalent-evaluate.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Check that if aria-brailleroledescription is not empty,
+ * the element has a non-empty aria-roledescription
+ * @memberof checks
+ * @return {Boolean}
+ */
+export default function brailleRoleDescriptionEquivalentEvaluate(
+  node,
+  options,
+  virtualNode
+) {
+  const brailleRoleDesc = virtualNode.attr('aria-brailleroledescription') ?? '';
+  if (!brailleRoleDesc.trim()) {
+    return true;
+  }
+  const roleDesc = virtualNode.attr('aria-roledescription');
+  if (typeof roleDesc !== 'string') {
+    this.data({ messageKey: 'noRoleDescription' });
+    return false;
+  }
+
+  if (roleDesc.trim() === '') {
```

Comment:
Should keep consistent use of if you're using `!brailleRoleDesc.trim()` or `brailleRoleDesc.trim() === '')`